### ISSUE TITLE
Highlightgroup `SpecialComment` not italic

### ISSUE THREAD

**Issue by rgnkn** [marko-cerovac/material.nvim#87]
First of all: Thank you for your phantastic cholorscheme. I guess it's my alltime favourite.

There is one minor thing that annoys me a bit. I use your italics setting for comments. But there is that inconsistency that SpecialComment remains non-italic.

Please check this Rust screenshot:

![Screenshot from 2022-04-05 23-23-19](https://user-images.githubusercontent.com/17874137/161852603-f1002d99-e1ea-42ee-8bb6-28b94e1825ec.png)

Here you can see that "normal" comments are italic (line 5) while special comments (Highlightgroup `SpecialComment`) remain non-italic (lines 1 and 3). If you're no Rust programmer those special comments are two distinct forms of "doc-comments" in Rust.

I guess that this isn't intended, so, I'd like to ask if it would be possibile to either extend the italic setting for comments to `SpecialComment` or to offer another field in the settings for italics.

Thx in advance.

**Comment by rgnkn** [comment]
Just one further question:

In case that I find more issues like this. Do you prefer me sending a request as above or would like to receive a PR?

Please inform.

**Comment by marko-cerovac** [comment]
This is one of those small things I overlooked when making the theme. Thanks for the input. It should be fixed now, and let me know if the issue still persists. It seems that treesitter doesn't have a highlight group for special comments. I fixed the default nvim ```SpecialComment``` highlight group to work with the italics.

As for the issue vs PR, which ever you like more. If you make an improvement or fix an issue yourself, feel free to send a PR, and if you don't feel like going trough the code, just open an issue and I'll come in and try to resolve it.
